pic-sure-auth-ui/src/test/javascript/headerSpec.js

```diff
@@ -89,18 +98,47 @@ define(["header/header", "picSure/userFunctions", "jquery","underscore"],
 						.callFake(function(object, callback){
 							callback({privileges: ['SUPER_ADMIN', 'SYSTEM','RESEARCHER','blahblah']});
 						});
+						applicationFunctionsSpy = spyOn(applicationFunctions, "fetchApplications").and
+						.callFake(function(object, callback){
+							callback([{}]);
+						});
 						header.View.render();
 						expect($('a[href="/psamaui/userManagement"][style="visibility: hidden"]', header.View.$el).length).toEqual(1);
 					});
 					// Since it is possible that admin users would not have access to picsure, perhaps this is the wrong approach
 					// That is not an issue related to the current ticket and those discussions will not be happening now.
-					it("shows the PIC-SURE UI button without needing any specific roles", function(){
+					// it("shows the PIC-SURE UI button without needing any specific roles", function(){
```

Comment:
unless PICSURE-UI comes by default everywhere, this should be removed.

Reply:
The PIC-SURE-UI does come by default everywhere moving forward